estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Estate Module",
+    "depends": ["base"],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/estate_property_offer_views.xml",
+        "views/estate_property_type_views.xml",
+        "views/estate_property_tag_views.xml",
+        "views/estate_property_views.xml",
+        "views/estate_menus.xml",
+    ],
+    "license": "LGPL-3",
```

Comment:
As a global comment: double quote `"` should be used for strings that will be displayed and single quote `'` for the rest
```suggestion
    'name': "Estate Module",
    'depends': ['base'],
    'data': [
        'security/ir.model.access.csv',
        'views/estate_property_offer_views.xml',
        'views/estate_property_type_views.xml',
        'views/estate_property_tag_views.xml',
        'views/estate_property_views.xml',
        'views/estate_menus.xml',
    ],
    'license': 'LGPL-3',
```